CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # authn.io ChangeLog
 
+## 7.0.1 - 2024-02-dd
+
+### Fixed
+- Load both manifest.json and manifest.webmanifest files.
```

Comment:
This should be a minor release ... I had some other suggestion for this that either I didn't submit or it got resolved away ... let me look for that.